open_lm/utils/llm_foundry_wrapper.py

```diff
@@ -50,5 +50,5 @@ def __init__(self, model, tokenizer):
             shift_labels=True,
         )
 
-    def generate(self, input_ids, **kwargs):
+    def generate(self, input_ids=None, **kwargs):
```

Comment:
maybe add `input_embeds` explicitly so the api is more transparent?

Reply:
Adding input_embeds created a bug in composer.... I grew older while finding it came from that T-T